packages/services/src/lib/trade-utils.ts

```diff
@@ -74,11 +74,12 @@ export function calculateSlippage(fromValueUSD: number): SlippageResult {
  * Major tokens exempt from trading constraint requirements (native tokens, established assets)
  * Populated from all token addresses configured in the system
  */
-export const EXEMPT_TOKENS = new Set([
-  // Collect all token addresses from all chains in config
-  ...Object.values(config.specificChainTokens).flatMap((chainTokens) =>
-    Object.values(chainTokens),
-  ),
-  // Add the zero address for native EVM tokens
-  "0x0000000000000000000000000000000000000000", // Ethereum, etc...
-]);
+export const EXEMPT_TOKENS = (specificChainTokens: SpecificChainTokens) =>
+  new Set([
```

Comment:
it looks like this is now constructing the set every time we want to check inclusion in the set.  Given this is used a bunch in `SimulatedTradeExecutionService`  in the code path of executing a trade, our hottest code path, it seems like we should try to make it so that this set only gets constructed once at startup rather than on every trade

Reply:
Good call. Fixing.